qemu/tests/cfg/virt_firmware_basic_test.cfg

```diff
@@ -0,0 +1,10 @@
+- virt_firmware_basic_test:
+    no Host_RHEL.m6 Host_RHEL.m7 Host_RHEL.m8 Host_RHEL.m9.u0 Host_RHEL.m9.u1
```

Comment:
`no i440fx` is make sense I think.

Reply:
Agree with you. Updated it, please help take a look at it. Many thanks.
Test result:
(1/1) Host_RHEL.m9.u2.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.virt_firmware_basic_test.q35: PASS (18.60 s)